recipe/meta.yaml

```
@@ -105,6 +107,14 @@ requirements:
 {% set tests_to_skip = tests_to_skip + " or test_clear_border_non_binary_out" %}     # [python_impl == 'pypy']
 {% set tests_to_skip = tests_to_skip + " or test_clear_border_non_binary_out_3d" %}  # [python_impl == 'pypy']
 
+# These seem to fail on cpython + linux_64 and seem generally flaky
+{% set tests_to_skip = tests_to_skip + " or test_subpix_border" %}
+{% set tests_to_skip = tests_to_skip + " or test_max_tree" %}
+{% set tests_to_skip = tests_to_skip + " or test_offsets_to_raveled_neighbors_explicit_0" %}   # [python_impl == 'cpython' ]
+{% set tests_to_skip = tests_to_skip + " or test_offsets_to_raveled_neighbors_explicit_1" %}   # [python_impl == 'cpython' ]
```

It looks like PyPy has the same test failures as CPython. Should we skip these on both?

```suggestion
{% set tests_to_skip = tests_to_skip + " or test_offsets_to_raveled_neighbors_explicit_0" %}   # [python_impl in ('cpython', 'pypy')]
{% set tests_to_skip = tests_to_skip + " or test_offsets_to_raveled_neighbors_explicit_1" %}   # [python_impl in ('cpython', 'pypy')]
```

---

Trying this change out to see if it resolves the remaining issue